product_customer_code_picking/models/stock_picking.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
As in https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming, please rename file to stock_move.py